### ISSUE TITLE
pip install from git setup.py TypeError < not supported for 'map's

### ISSUE THREAD

**Issue by howff** [ckan/ckan#4377]
### CKAN Version if known (or site URL)

2.8.1 source install from git into a fresh install of Ubuntu 18.04 (x86_64)
(same result with 2.9.0a)
(setuptools is v40.0.0, same result downgrading to v36.1)

### Please describe the actual behaviour

Installing using the 2.8 documents:
pip install -e 'git+https://github.com/ckan/ckan.git@ckan-2.8.1#egg=ckan'
File "/usr/lib/ckan/default/src/ckan/setup.py", line 34,
if parse_version(setuptools_version) < min_setuptools_version:
    TypeError: '<' not supported between instances of 'map' and 'map'

Is this because I am using Python 3? Python 2 is not installed.

**Comment by torfsen** [comment]
Currently only Python 2 is supported. Please try again using Python 2 to check whether this solves your problem.

**Comment by scoutinhoGit** [comment]
Hi,
I have followed the more recent instructions to install CKAN:

https://docs.ckan.org/en/latest/maintaining/installing/install-from-source.html

I have depared with the same issue of  "TypeError: '<' not supported between instances of 'map' and 'map'"

I understand the explanation of "Currently only Python 2 is supported."

But today Python 2 is depreciated. So, is there any stable version to CKAN for python3?  Or we can only install a estable version 2.8 using  Python 2? 

Regards,

Sergio
